Doc/c-api/unicode.rst

```diff
@@ -1396,18 +1396,28 @@ They all return ``NULL`` or ``-1`` if an exception occurs.
    :c:func:`PyErr_Occurred` to check for errors.
 
 
-.. c:function:: int PyUnicode_EqualToUTF8(PyObject *unicode, const char *string)
+.. c:function:: int PyUnicode_EqualToUTF8AndSize(PyObject *unicode, const char *string, Py_ssize_t size)
 
-   Compare a Unicode object with a UTF-8 or ASCII encoded C string
-   and return true (``1``) if they are equal, or false (``0``) otherwise.
-   If the Unicode object contains null or surrogate characters or
+   Compare a Unicode object with a char buffer which is interpreted as
+   being UTF-8 or ASCII encoded and return true (``1``) if they are equal,
+   or false (``0``) otherwise.
+   If the Unicode object contains surrogate characters or
    the C string is not valid UTF-8, false (``0``) is returned.
 
    This function does not raise exceptions.
 
    .. versionadded:: 3.13
 
 
+.. c:function:: int PyUnicode_EqualToUTF8(PyObject *unicode, const char *string)
+
+   Similar to :c:func:`PyUnicode_EqualToUTF8AndSize`, but compute the string
```

Comment:
Since both arguments are string, I suggest to use italic to show that you're talking about the second argument :-)

```suggestion
   Similar to :c:func:`PyUnicode_EqualToUTF8AndSize`, but compute *string*
```

or:

```
the *string* string 
```